content/en/platform/corda/4.6/enterprise/cordapps/cordapp-overview.md

```diff
@@ -21,7 +21,7 @@ Corda node owners can invoke over RPC:
 {{< figure alt="node diagram" width=80% zoom="../resources/node-diagram.png" >}}
 
 {{< note >}}
-When designing your CorDapp, you may want to consider using the [CorDapp Design Language](/docs/cdl/cdl/cdl-overview) to structure and organise your designs. This can be especially helpful for complex CorDapps.
+When designing your CorDapp, you may want to consider using the [CorDapp Design Language](/../../../../../tools/cdl/cdl-overview.md) to structure and organise your designs. This can be especially helpful for complex CorDapps.
```

Comment:
```suggestion
When designing your CorDapp, you may want to consider using the [CorDapp Design Language](../../../../../tools/cdl/cdl-overview.md) to structure and organise your designs. This can be especially helpful for complex CorDapps.
```